package.json

```diff
@@ -31,28 +31,28 @@
         "test": "jest"
     },
     "dependencies": {
-        "@lwc/compiler": "2.2.3",
-        "@lwc/engine-dom": "2.2.3",
+        "@lwc/compiler": "2.2.6",
+        "@lwc/engine-dom": "2.2.6",
         "@lwc/jest-preset": "11.0.1",
         "@lwc/jest-resolver": "11.0.1",
         "@lwc/jest-serializer": "11.0.1",
         "@lwc/jest-transformer": "11.0.1",
-        "@lwc/module-resolver": "2.2.3",
-        "@lwc/synthetic-shadow": "2.2.3",
-        "@lwc/wire-service": "2.2.3",
+        "@lwc/module-resolver": "2.2.6",
+        "@lwc/synthetic-shadow": "2.2.6",
+        "@lwc/wire-service": "2.2.6",
         "@salesforce/wire-service-jest-util": "4.0.0",
         "chalk": "~4.1.1",
-        "fast-glob": "^3.2.5",
-        "jest": "26.6.3",
+        "fast-glob": "^3.2.7",
+        "jest": "27.0.6",
```

Comment:
FYI, this will require a major version bump.

Reply:
Is there any way to support both jest 26 and 27, like we do [in lwc-test](https://github.com/salesforce/lwc-test/blob/d278222f92bbc622ac0f35e05c403c1ac10ccbb4/.circleci/config.yml#L3)?

Reply:
Packages in `lwc-test` are jest plugins while `sfdx-lwc-jest` is a jest wrapper. I don't think it's possible with the way sfdx-lwc-jest is set up to distribute multiple versions of jest.

An alternative approach would be to externalize `jest` dependency and require all the SFDX projects to install `jest` as a peer dependency. In general Salesforce developers aren't familiar enough with the NodeJS ecosystem and NPM to deal with peer dependencies. Based on my experience Salesforce developers are looking for packaged and easy-to-install solutions.

@nolanlawson Are you concerned about the potential breaking changes jest@27 introduces?

Reply:
@pmdartus Based on [the release notes](https://github.com/facebook/jest/releases/tag/v27.0.0) it seems to me that there are things that can start breaking in v27. As long as we release a major version bump we should be fine. I didn't realize this was a wrapper and not a plugin.